client/my-sites/site-settings/site-icon-setting/style.scss

```diff
@@ -1,9 +1,15 @@
-.form-label.site-icon-setting__heading {
+.site-settings .form-label.site-icon-setting__heading {
 	display: flex;
 	align-items: center;
+	justify-content: normal;
+
+	@include breakpoint-deprecated( ">660px" ) {
```

Comment:
Why not fix it in its original location?

https://github.com/Automattic/wp-calypso/blob/d440a6f9ddcf3a92e7f8b08421a6e2d8d54e5ca0/client/my-sites/site-settings/style.scss#L199-L204